### PR TITLE
Turkish localization updates

### PR DESCRIPTION
**Pull request by mustafau**
Various fixes and additions to the Turkish translations.

### Testing done

Screen Shot 2023-02-20 at 01 58 15

Screen Shot 2023-02-20 at 02 01 36

### Proposed changelog entries

- […]

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7662"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/model/AbstractItem/confirm-rename_tr.properties
Comment (NotMyFault) [inline core/src/main/resources/hudson/model/AbstractItem/confirm-rename_tr.properties:1]:
```suggestion
DescribeRename={1} {0} sinin Adını Değiştir
```

File: core/src/main/resources/lib/hudson/project/configurable_tr.properties
Comment (NotMyFault) [inline core/src/main/resources/lib/hudson/project/configurable_tr.properties:23]:
```suggestion
delete={0} yi Sil
```

**Review by NotMyFault — COMMENTED** [review]
Thanks for the PR, I left some feedback for consideration.

File: core/src/main/resources/hudson/model/AbstractItem/confirm-rename_tr.properties
Comment (mustafau) [reply core/src/main/resources/hudson/model/AbstractItem/confirm-rename_tr.properties:1]: "si" and "nin" are suffixes in Turkish which needs to be appended to the word without space. The resulting word should be "Proje**sinin**" not "Proje sinin"

File: core/src/main/resources/lib/hudson/project/configurable_tr.properties
Comment (mustafau) [reply core/src/main/resources/lib/hudson/project/configurable_tr.properties:23]: Same as the others. The resulting word should be "Proje**yi**" not "Proje yi"

**Review by NotMyFault — APPROVED** [review]
Thanks!

File: core/src/main/resources/hudson/model/AbstractItem/confirm-rename_tr.properties
Comment (NotMyFault) [reply core/src/main/resources/hudson/model/AbstractItem/confirm-rename_tr.properties:1]: Thanks for the clarification!

**Comment by mustafau** [comment]
Thank you

**Review by timja — APPROVED** [review]

File: core/src/main/resources/lib/hudson/rssBar_tr.properties
Comment (timja) [inline core/src/main/resources/lib/hudson/rssBar_tr.properties:25]: are these brackets needed? the english version doesn't have them

Comment (mustafau) [reply core/src/main/resources/lib/hudson/rssBar_tr.properties:25]: It's currently not possible to have grammatically correct translation for these strings. Using parenthesis I was able to build correct representation. 

I'm pretty satisfied with the result however if wanted I can work on a follow up PR refactoring original strings to allow better localization.

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
